src/frontend/web_application/src/services/api-client/index.js

```diff
@@ -24,16 +28,54 @@ if (BUILD_TARGET === 'server') {
   };
 }
 
-export default function getClient() {
+const buildClient = () =>
+  axios.create({
+    baseURL: getBaseUrl(),
+    responseType: 'json',
+    headers,
+    paramsSerializer: params => queryStringify(params, headers),
+    transformRequest: ([(data) => {
+      if (data instanceof Blob) {
+        if (typeof FormData !== 'function') {
+          throw new Error('not a browser environment');
+        }
+
+        const transformedData = new FormData();
+        transformedData.append('attachment', data);
```

Comment:
I'm not sure it is a good idea to put here the attachment (the term is related to message's attachment). 

Importing contacts might not be named the same.

Reply:
I put it in a specific object, to be able to carry the field name with it all the way to the transformer. Object which also handle transformation to formdata and the browser env check that comes with it.